server/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+
+const controllers = require('./controllers');
+
+const app = express();
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'client', 'build')));
+
+app.use(controllers);
+app.get('/', (req, res) => {
```

Comment:
if you use `app.get('*',` instead of `'/'` you can forward all unhandled routes to React for routing